package.json

```diff
@@ -247,6 +247,7 @@
 		"@types/glob": "^7.1.1",
 		"@types/mocha": "^7.0.2",
 		"@types/node": "^16.11.7",
+		"@types/selenium-webdriver": "^4.1.22",
```

Comment:
what are the `@types/selenium-webdriver` and `selenium-webdriver` dependencies needed for?

Reply:
`selenium-webdriver` automates browser tasks like clicking buttons, filling out forms, navigating between pages, and more.
`@types/selenium-webdriver` enhances code editor features like auto-completion, type-checking, and inline documentation.

I removed both and tried running the build, but it failed with the following error.

```
ERROR in /liberty-tools-vscode/src/test/GradleInitProject.ts(2,11)
      TS2305: Module '"vscode-extension-tester"' has no exported member 'WebDriver'.

ERROR in liberty-tools-vscode/src/test/utils/macUtils.ts(12,16)
      TS2339: Property 'click' does not exist on type 'DefaultTreeItem'.

ERROR in /liberty-tools-vscode/src/test/utils/macUtils.ts(22,21)
      TS2339: Property 'click' does not exist on type 'InputBox'.

ERROR in /liberty-tools-vscode/src/test/utils/testUtils.ts(46,17)
      TS2339: Property 'click' does not exist on type 'InputBox'.

ERROR in /liberty-tools-vscode/src/test/utils/testUtils.ts(138,23)
      TS2339: Property 'getDriver' does not exist on type 'Workbench'.

ERROR in /liberty-tools-vscode/src/test/utils/testUtils.ts(161,23)
      TS2339: Property 'getDriver' does not exist on type 'Workbench'.

ERROR in /liberty-tools-vscode/src/test/utils/testUtils.ts(179,17)
      TS2339: Property 'click' does not exist on type 'InputBox'.

8 errors have detailed information that is not shown.
Use 'stats.errorDetails: true' resp. '--stats-error-details' to show it.

webpack 5.85.0 compiled with 8 errors and 2 warnings in 1345 ms


```

I then added @types/selenium-webdriver: ^4.1.22 back to the devDependencies, and now the build starts running.